Jenkinsfile

```
@@ -54,7 +54,7 @@ pipeline {
     stage('Run GIS interface server'){
       steps{
         script{
-            sh 'scp src/test/resources/IUDX-GIS-SERVER.postman_collection.json jenkins@jenkins-master:/var/lib/jenkins/iudx/gis/Newman/'
+            sh 'scp src/test/resources/IUDX_GIS_Server_APIs(v3.5.0).postman_collection.json jenkins@jenkins-master:/var/lib/jenkins/iudx/gis/Newman/'
```

```suggestion
            sh 'scp src/test/resources/IUDX_GIS_Server_APIs\(v3.5.0\).postman_collection.json jenkins@jenkins-master:/var/lib/jenkins/iudx/gis/Newman/'
```
need to escape the parentheses